addons/l10n_hu_edi/tests/common.py

```diff
@@ -153,7 +151,7 @@ def create_invoice_simple(self):
         })
 
     def create_advance_invoice(self):
-        """ Create a sale order, an advance invoice and a final invoice. """
+        """ Create a sale order, and an advance invoice. """
```

Comment:
A really small nothing but since I asked about other stuff, I'll add this too, sorry haha :smile:
```suggestion
        """ Create a sale order and an advance invoice. """
```